src/components/Inputs/CheckboxInput/CheckboxInput.js

```diff
@@ -75,7 +77,9 @@ export const CheckboxInput = ({
         ev.target.type === 'checkbox' ? ev.target.checked : ev.target.value
       doValidation(!val, touched)
       setIsChecked(!val)
-      setFieldTouched(true)
+      if (!!setFieldTouched) {
```

Comment:
This only gets called when we're hitting the field from Form.js e.g. if we use it "directly" this will likely not be defined